webcompat/templates/home-page/issue-wizard-form.html

```diff
@@ -351,13 +352,43 @@ <h3 class="page-heading">Filing a web compatibility issue</h3>
             Good job! You can&nbsp;<strong>report via GitHub</strong>&nbsp;or&nbsp;<strong>report anonymously</strong>.
           </div>
           <div class="row">
-            <button class="button button-primary right js-Button" id="submitgithub" type="submit" name="github-auth-report">
+            <button class="button button-secondary right js-Button" id="submitgithub" type="submit" name="github-auth-report">
+              <img class="" src="{{ url_for('static', filename='img/svg/icons/svg-github-white.svg') }}" title="Report via GitHub" alt="Report via GitHub" />
               {% if session.username %}Report as {{ session.username }}{% else %}Report via GitHub{% endif %}
             </button>
-            <button class="button button-secondary js-Button" id="submitanon" type="submit" name="github-proxy-report">
+            <button class="button button-primary" id="open-username" type="submit">
```

Comment:
I'm a little confused here.Removing the `name` here will break the reporting backend:

https://github.com/webcompat/webcompat.com/blob/04e3e5d5116ab8fffb7c09aaacb79c33ff4940f9/webcompat/issues.py#L27-L33

But I see below:

```
<button class="button button-primary js-Button" id="submitanon" type="submit" name="github-proxy-report">
                Send report
</button>
```

Is this correct? I think if you click "Report Anonymously" it will break, and I'm not sure what "Send report" is for, but it will report anonymously.

Reply:
@miketaylr, the Report Anonymously button doesn't submit the form. It reveals an input where the user can type the optional username and only then the form gets submitted (the username field can be left empty). The "real" button that submits the form anonymously is the one below ("Send report").

Reply:
OK, thanks!